3684/1/table_maker.rb

```diff
@@ -0,0 +1,15 @@
+require 'terminal-table'
+
+class TableMaker
+  TABLE_HEADIN = ['gem name', 'used by', 'contributors', 'issues', 'stars', 'watch', 'forks'].freeze
+
+  def initialize(gems_info)
+    @info = gems_info
+  end
+
+  def make_table
+    rows = []
+    @info.select { |item| rows << item.values }
```

Comment:
`row = @info.select { |item| item.values }`
but in this particular case `rows = @info.map(&:values)` should be enough